python/dataproc_templates/snowflake/README.md

```diff
@@ -0,0 +1,133 @@
+## 1. Snowflake To GCS
+
```

Comment:
Add Introduction:
Template for reading data from a Snowflake table or custom query and writing Google Cloud Storage. It supports writing JSON, CSV, Parquet and Avro formats.